tests/test_model.py

```diff
@@ -522,3 +522,417 @@ def test_model_diskonlyfilemodel_field_is_constructed_correctly(
         assert relevant_field == input
     else:
         assert relevant_field.filepath == input
+
+
+class TestFmComponentProcessIntegrationWithDimr:
+    def get_fm_dimr_config_data(self, input_data_process):
+        dimr_config_data = f"""<?xml version="1.0" encoding="utf-8"?>
```

Comment:
no need to first put it into a variable, you can immediately return the string :)

Reply:
True, I put it like this during testing because I was debugging through it, will update.